CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Contributing
+
+Thank you for your interest in making [FerretDB](https://github.com/FerretDB/FerretDB) better!
+
+## Finding something to work on
+
+We are interested in all contributions, big or small, in code or documentation.
+But unless you are fixing a very small issue like a typo,
+we kindly ask you first to [create an issue](https://github.com/FerretDB/github-actions/issues/new),
+to leave a comment on an existing issue if you want to work on it,
+or to [join our Slack chat](https://github.com/FerretDB/FerretDB/README.md#community) and leave a message for us there.
+This way, you will get help from us and avoid wasted efforts if something can't be worked on right now
+or someone is already working on it.
+
+You can find a list of good issues for first-time contributors [there](https://github.com/FerretDB/github-actions/contribute).
+
+## Setting up the environment
+
+### Requirements
+
+The only way to run github-actions is to run them remotely on the repository.
+
+You will need Go 1.19 or later on the host.
+If your package manager doesn't provide it yet,
+please install it from [go.dev](https://go.dev/dl/).
+
+You will also need `git` installed; the version provided by your package manager should do.
```

Comment:
I'm not sure we really need to mention git here... Like, people also need GitHub accounts to contribute, but we don't mention it :)

Reply:
It was inspired by the FerretDB's contributing guidelines

Reply:
I think we actually added it because it wasn't clear for someone 😁

Reply:
Hmmm... Ok then :)